system-int-test/src/cmr/system_int_test/system.clj

```diff
@@ -41,7 +41,7 @@
              ;; A map of the components (echo, elastic, db, and message queue) to whether they are
              ;; in-memory or external
              :component-type-map component-type-map}]
-    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :urs :ingest])))
+    (transmit-config/system-with-connections sys [:echo-rest :search :access-control :metadata-db :urs :ingest])))
```

Comment:
I had a comment for Erich in his logging PR that we shouldn't really use the transmit library for our system integration tests. Someone could change an endpoint and no tests would break to let us know we broke backwards compatibility. Rather we should use black box HTTP calls (using the url-helper namespace) for our integration tests.

What do you think about making http requests rather than using the `cmr.transmit.association` and `cmr.transmit.services` namespaces?

Reply:
I agree. This is a temporary fix to use metadata-db rather than search-app for the verification. We will change it back to search when we added the search by service capability in search (probably in two weeks). I say we leave it as is for now. The existing search function is using clj-http directly. BTW, this is documented in the docstring of `cmr.system-int-test.utils.service-util/assert-service-associated-with-query`.

Reply:
I smell a CMR client coming ... ;-)

Hey, so we should probably put a comment here linking to a ticket if this is going to be temporary ...

Reply:
OK. Changed the function to use clj-http directly and referenced CMR-4280 in comment.